eZ/Bundle/EzPublishCoreBundle/Command/CheckURLsCommand.php

```diff
@@ -55,8 +55,8 @@ public function __construct(
 
     public function configure(): void
     {
+        $this->setAliases(['ezplatform:check-urls']);
```

Comment:
Is there any reasons why you are not using `getDeprecatedAliases` here?

Reply:
It was fixed in one of the --fixup commit :wink: See https://github.com/ezsystems/ezplatform-kernel/pull/153/files#diff-50f5e7b9f0e7bbd5c417eb7f7f6762413a9527fb3a709d4d978939f0db41595eR122-R125